tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/external/iso_3166_status.txt

```diff
@@ -30,15 +30,15 @@ BL	;	Officially assigned	;	Saint-Barthélemy
 BM	;	Officially assigned	;	Bermuda
 BN	;	Officially assigned	;	Brunei Darussalam
 BO	;	Officially assigned	;	Bolivia (Plurinational State of)
-BQ	;	Officially assigned	;	Bonaire, Sint Eustatius and Saba
+BQ	;	Formerly used	;	Bonaire, Sint Eustatius and Saba
```

Comment:
Similar; "The code BQ was formerly used for British Antarctic Territory (BQ, ATB, --). ... BQ has been reused and reassigned to Bonaire"

Reply:
reverted